optimize/upgrade/pom.xml

```diff
@@ -188,6 +188,10 @@
       <version>2.12.0</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+    </dependency>
```

Comment:
❌ This change broke building with `mvnd` (Maven Daemon)

```
mvnd clean install -Dquickly -Dskip.fe.build -Dspotless.skip
[INFO] Processing build on daemon 60b1894c
[INFO] Scanning for projects...
[ERROR] Some problems were encountered while processing the POMs
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project io.camunda.optimize:upgrade-optimize:8.7.0-SNAPSHOT (/Users/korthout/gh/camunda/camunda/optimize/upgrade/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.apache.commons:commons-lang3:jar -> version 3.14.0 vs (?) @ line 191, column 5
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

@buccarel Can you revert this change?